### ISSUE TITLE
[Accessibility]: Signpost content is not read by screen-reader when opened

### ISSUE THREAD

**Issue by Tushar2098** [vmware/clarity#4389]
## Describe the bug

Signpost content is not read by NVDA/Voiceover when opened. As of now, it just announces expand/collapse state of signpost

I see there is already an issue https://github.com/vmware/clarity/issues/3523 created for the same..but it is closed.

## How to reproduce

1. Go to https://v2.clarity.design/signposts
2. Click on the info icon to open the signpost
3. Observe signpost content is not announced by screen-reader

## Expected behavior
Screen-reader should announce the signpost content when opened. 

## Versions
This issue is also reproducible in latest Clarity version.

**Comment by bashibozuk** [comment]
Are you guys planning to invest efforts on this issue any time soon ?

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
